youtube_dl/extractor/amc.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    smuggle_url,
+    url_basename
+)
+
+
+class AMCIE(InfoExtractor):
+    IE_NAME = 'amc'
+    _VALID_URL = r'https?://www\.amc\.com/.*?'
```

Comment:
You should use a stricter regex and also get the `display_id`, e.g. something like `https?://(?:www\.)?amc\.com/shows/[^/]+/(?P<display_id>[a-z0-9]+)`